Portal_Application/Startup.cs

```diff
@@ -17,18 +18,14 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
-        // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            string connectionString = "Server=(localdb)\\mssqllocaldb;Database=productsdb;Trusted_Connection=True;";
+            services.AddDbContext<ApplicationContext>(options => options.UseSqlServer(connectionString));
             services.AddControllersWithViews();
```

Comment:
I don't think you need `AddControllersWithViews`, `AddControllers` will be enough